src/universal/modules/teamDashboard/containers/TeamArchive/TeamArchiveRoot.js

```diff
@@ -35,7 +35,7 @@ const TeamArchiveRoot = ({atmosphere, match, team}) => {
         return (
           <TransitionGroup appear component={null}>
             {error && <ErrorComponent height={'14rem'} error={error} />}
-            {renderProps &&
+            {renderProps && team &&
```

Comment:
When would `team` be falsy? Seems like a for the `TeamArchiveRoot` this should never be the case.

Reply:
team is fed as a prop, which comes from `TeamRoot` We have 3 choices: wait for TeamRoot to return from the server before we ask for `TeamArchiveRoot`, build `TeamArchiveRoot` to handle a null team, or ignore the data from the parent & request it again inside `TeamArchiveRoot`. Waiting sucks and requesting the same info twice sucks, so I chose the second option. See #1371 for more detail